monai/utils/misc.py

```diff
@@ -217,3 +218,39 @@ def set_determinism(
         torch.backends.cudnn.benchmark = False
     else:
         torch.backends.cudnn.deterministic = False
+
+
+def list_to_dict(items):
+    """
+    To convert a list of "key=value" pairs into a dictionary.
+    For examples: items: `["a=1", "b=2", "c=3"]`, return: {"a": "1", "b": "2", "c": "3"}.
+    If no "=" in the pair, use None as the value, for example: ["a"], return: {"a": None}.
+
+    """
+
+    def _parse_var(s):
+        items = s.split("=")
```

Comment:
Here you can say `s.split("=",1)` to split by = character only once, this would return 1 or 2 values exactly.

Reply:
updated in the latest commit.
Thanks.